pkg/globalnet/controllers/ipam/gatewaymonitor.go

```diff
@@ -209,6 +216,10 @@ func (gm *GatewayMonitor) processNextEndpoint() bool {
 			gm.syncMutex.Unlock()
 		} else {
 			klog.V(log.DEBUG).Infof("We are on non-gatewayNode. GatewayNode ip is %s", endpoint.Spec.PrivateIP)
+			if gm.mtuConfig.successfullyRead {
```

Comment:
Is this to restore?, What if the pod is restarted when the settings are configured already?

Shouldn't we remove also on pod stop?

BTW, I'm not sure if it's worth bothering with the removal of this.

Reply:
Yes @mangelajo this is to restore the old values when the Gateway migrates to a new node. 
TBH from what I understand there does not seem to be any harm even if we leave it, but just though of getting some feedback and included this change.

So, are you suggesting that we can skip it?

Reply:
I would simplify the code by avoiding the cleanup as we do for similar other parameters.